lumi-checks/microbenchmarks/mpi/osu/osu_tests.py

```diff
@@ -0,0 +1,315 @@
+# Copyright 2016-2022 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+from hpctestlib.microbenchmarks.mpi.osu import (osu_latency,
+                                                osu_bandwidth,
+                                                build_osu_benchmarks)
+
+
+class build_osu_benchmarks_gpu(build_osu_benchmarks):
+    @run_before('setup')
+    def set_build_mode(self):
+        if self.current_system.name in ['lumi']:
+            self.build_locally = False
+            self.build_system.cc = 'hipcc'
+            self.build_system.cxx = 'hipcc'
+            self.build_system.cflags = ['-I/opt/cray/pe/mpich/8.1.8/ofi/crayclang/10.0/include']
+            self.build_system.ldflags = ['-L/opt/cray/pe/mpich/8.1.8/ofi/crayclang/10.0/lib', '-lmpi', '-L/opt/cray/pe/mpich/8.1.8/gtl/lib', '-lmpi_gtl_hsa']
+            self.build_system.cppflags = ['-D__HIP_PLATFORM_AMD__']
```

Comment:
Would it make sense something like what suggested here https://github.com/Lumi-supercomputer/reframe/pull/4#discussion_r837542428?

Reply:
Added to #3.